DOCUMENTATION.md

```diff
@@ -89,14 +89,24 @@ or use `pm2` by downloading it from npm - `npm i -g pm2` and start the module by
 The repo detail view also provides features for performing the following git operations,
 ```
 - Adding a new branch
+- List all branches including remote branches
+- Switching branches
 - Pulling changes from remote
+- Branch deletion 
 - Fetching changes from remote
 - Adding a new remote repo
 ```
-![repo-card-details](https://user-images.githubusercontent.com/65342122/87232642-1f175400-c3de-11ea-8ead-80cd5ab4c37c.png)
+![Repo view detail](https://user-images.githubusercontent.com/65342122/87664834-8ee16200-c783-11ea-885e-6a1e29626a47.png)
 
+- "List all branches" under available branches, lists all the branches from the and remote. Click on a branch to checkout to that branch. 
+- Can delete local branches and cannot be retrievd again( please be catious! )
```

Comment:
If the branches are in remote, then it can be restores. So please refactor this line. Also there is a spelling error for 'cautious'